pipelined/bevy_core_pipeline/src/lib.rs

```diff
@@ -144,11 +150,43 @@ impl Plugin for CorePipelinePlugin {
             .unwrap();
         graph.add_sub_graph(draw_3d_graph::NAME, draw_3d_graph);
 
+        graph.add_node(node::HDR_TEXTURE, HdrTextureNode::default());
         graph.add_node(node::MAIN_PASS_DEPENDENCIES, EmptyNode);
         graph.add_node(node::MAIN_PASS_DRIVER, MainPassDriverNode);
         graph
             .add_node_edge(node::MAIN_PASS_DEPENDENCIES, node::MAIN_PASS_DRIVER)
             .unwrap();
+        graph
```

Comment:
For rafx, SDR / HDR is configurable. If HDR has some significant performance impact somewhere then I suppose it would make sense to make it configurable.